src/container/container_iv.c

```diff
@@ -497,12 +534,23 @@ cont_iv_ent_update(struct ds_iv_entry *entry, struct ds_iv_key *key,
 						civ_ent->iv_snap.snap_cnt);
 			if (rc)
 				return rc;
+		} else if (entry->iv_class->iv_class_id ==
+						IV_CONT_AGG_EPOCH_REPORT) {
+			rc = cont_iv_ent_agg_eph_update(entry, key, src);
+			if (rc)
+				return rc;
+		} else if (entry->iv_class->iv_class_id ==
+						IV_CONT_AGG_EPOCH_BOUNDRY) {
+			rc = cont_iv_ent_agg_eph_refresh(entry, key, src);
```

Comment:
There are two use cases here.  both will use IV update. 
1. every nodes report its EC epoch to the leader.
2. Leader sync (broadcast) its minum epoch to all of servers.

Yes, we can use single IV class ID, but it will make iv callback itself a bit complicate. since iv class ID is cheap anyway. so I choose to use two class ID to make it clearer inside the callback. (easy to follow).  i.e. I would rather use different class ID, instead of tweak these inside flags, which is already complicate enough. 

IV_REPORT means each node report its EC epoch to the leader.
IV_BOUNDRY means server tell each node the current vos aggregation epoch boundary.
Sure I will add some comments.